operator/v1/types_network.go

```diff
@@ -349,6 +353,35 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type PolicyAuditConfig struct {
+	// RateLimit is the approximate maximum number of messages to generate per-second per-node. If
+	// unset the default of 20 msg/sec is used.
+	// +optional
+	RateLimit *uint32 `json:"rateLimit,omitempty"`
+
+	// MaxFilesSize is the max size an ACL_audit log file is allowed to reach before rotation occurs
+	// Default is 50MB
+	// +optional
+	MaxFileSize *uint32 `json:"maxFileSize,omitempty"`
+
+	// Messages are output in syslog format. Destination is the destination for policy log messages.
+	// Regardless of this config logs will always be dumped to ovn at /var/log/ovn/ however
+	// you may also configure additional output as follows.
+	// Messages are output in syslog format.
+	// Valid values are:
+	// - "libc" -> to use the libc syslog() function of the host node's journdald process
+	// - "udp:host:port" -> for sending syslog over UDP
+	// - "unix:file" -> for using the UNIX domain socket directly
+	// - "null" -> to discard all messages logged to syslog
+	// The default is "null"
+	// +optional
+	Destination string `json:"destination,omitempty"`
+
+	// SyslogFacility the RFC5424 facility for generated messages, e.g. "kern". Default is "local0"
+	// +optional
+	SyslogFacility string `json:"syslogFacility,omitempty"`
```

Comment:
as these are known, please add an enum validation and use an explicit default.

Reply:
Make it required then of course.

Reply:
This is optional, and only configured if the syslog `Destination` is set to something besides "null", again default is set by CNO

Reply:
Add regex validation, Let me know if this works, and thanks for the review @sttts!